File: bee_1387.py
def main():
    # Inicializar
    left = -1
    right = -1
    
    # Processar
    while left != 0 and right != 0:
        left, right = list(map(int, input().split()))
        
        # Saída
        if left != 0 and right != 0:
            print(left + right)
main()

File: bee_1387.c
#include <stdio.h>

int main() {
    int left = -1;
    int right = -1;
    
    while (left != 0 && right != 0) {
        scanf("%d %d", &left, &right);
        
        if (left != 0 && right != 0) {
            printf("%d", left + right);
        }
    }
}